docs/monitoring/cluster-alerts.asciidoc

```diff
@@ -50,4 +50,6 @@ To receive email notifications for the Cluster Alerts:
 {stack-ov}/actions-email.html#configuring-email[Configuring Email Accounts].
 2. Configure the `xpack.monitoring.cluster_alerts.email_notifications.email_address` setting in `kibana.yml` with your email address.
 
+NOTE: If you're using a dedicated monitoring cluster this setting needs to be set in the cluster thats being monitored, and not in the cluster thats monitoring it.
```

Comment:
Since this is a `kibana.yml` setting, I wouldn't talk about setting it in a cluster. That could be interpreted as setting it in the Elasticsearch cluster. 

See step 5 here: https://www.elastic.co/guide/en/elastic-stack-overview/current/monitoring-production.html. I'd use terminology from there over here as well to indicate that the user must set this setting in the Kibana instance that's connected to the production cluster and not the dedicated Kibana instance that's connected to the monitoring cluster.

Reply:
I've drafted that change and pushed the commit.  If it's not what you had in mind, @ycombinator let me know.  Otherwise, LGTM and builds successfully.

Reply:
Thanks for the edit @lcawl. LGTM!